src/compiler/common/Identifier.ts

```diff
@@ -37,6 +37,12 @@ export class Identifier extends IdentifierBase<ts.Identifier> {
         return this.global.languageService.getDefinitions(this);
     }
 
+    /**
+     * Gets the name of what this identifier is referencing.
+     */
+    getName(): string {
```

Comment:
This duplicates what `getText()` does, right? I'd rather not have multiple methods for the same thing.